internal/runtime/source.go

```diff
@@ -11,7 +11,7 @@ func init() {
 
 // SayHello says hello to the specified name, using
 // the saved greeting
-func SayHello(name string) string {
+func SayHello(cur realm, name string) string {
```

Comment:
Why does this need to be a crossing func?

Reply:
I think it is neccessary for crossrealm implementation,
it seems like when having it without the modification we have
`/std.InternalError"�-recovered: wrong number of arguments in call to SayHello: want 1 got 2`

this is because when doing exec call we're passing currentRealm realm as first parameter of the call